### PR TITLE
Update to Readme

### PR DESCRIPTION
**Pull request by aberrey**
This is an update to the readme with more information about GAEN and a more expansive explanation of the Safe Paths project. Please review for typos!!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,47 +9,66 @@
...
-[Apps Gone Rogue: Maintaining Personal Privacy in an Epidemic](https://drive.google.com/file/d/1nwOR4drE3YdkCkyy_HBd6giQPPhLEkRc/view?usp=sharing)
+### Custom Builds 
 
-### Downloads for COVID Safe Paths
+We welcome public health authorities and other organizations implementing digital contact tracing strategies to create custom builds for their specific needs, incorporate Safe Paths features into their applications, or create downstream projects that stay linked to the Safe Paths project. 
 
-[Google Play](https://play.google.com/store/apps/details?id=org.pathcheck.covidsafepaths) | [Apple Store](https://apps.apple.com/us/app/covid-safe-paths/id1508266966)
+### GAEN Instances
+
+The rules for GAEN require that each public health authority release their own application. The goal with Project Aurora is to suppor this deployment strategy.
```
Comment (alpita-masurkar) [inline README.md]: typo-- support*

File: README.md [hunk 2]
```diff
@@ -9,47 +9,66 @@
...
+
+Safe Paths is designed to work with Safe Places, a tool for contact tracing teams to work with location data in the course of contact tracing interviews and to publish points of concern. Without having to sharing their own location history, a Safe Paths user can download the points of concern from their PHA to identify if they have had risk of an exposure. 
+
+Project Aurora supports any GAEN compliant Key Server for exposure notification. 
 
-### Downloads for Private Kit (technology beta)
+## Broad Non-Developer Community 
 
-[Google Play](https://play.google.com/store/apps/details?id=edu.mit.privatekit) | [Apple Store](https://apps.apple.com/us/app/private-kit-prototype/id1501903733)
+One of the important aspects of the Safe Paths open source project is that it's supported by a large community of volutneers in addition to the open source developer community. More than 1,400 Path Check volunteers are working together to support Safe Paths and help drive adoption around the world.
```
Comment (alpita-masurkar) [inline README.md]: typo-- volunteers*

File: README.md [hunk 3]
```diff
@@ -9,47 +9,66 @@
...
 
-Location logs provide time-stamped records of where users have been, allowing them to share information with health officials accurately and quickly. This helps support contact tracing efforts to slow the spread of the virus.
+## Privacy Preserving 
 
-What’s truly special about Safe Paths, though, is its privacy protection.
+What’s truly special about Safe Paths is our strong commitment to preserving the privacy of individual users. We're building an application that can help contain outbreaks of COVID-19 without forcing users to sacrafice their personal privacy. For example, if a user chooses to use Safe Paths to maintain a record of their locations use a time stamped GPS log, which stores 14 days of data in 5 minute increments. The location log is stored on the users phone. The logged data only leave the device if the user chooses to send the information to an authorized public health authority (PHA) as part of the contact tracing prcoess.
```
Comment (alpita-masurkar) [inline README.md]: typo-- sacrifice*

**Review by alpita-masurkar — CHANGES_REQUESTED** [review]
a few typos... left some comments

File: README.md (hunk 3 above)
Comment (tstirrat) [inline README.md]: The logged data only leave[s] the device if the user chooses to send the information to an authorized public health authority (PHA) as part of the contact tracing [process].

**Review by tstirrat — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -9,47 +9,66 @@
...
 
-Data never leaves a user's device without their password entry and explicit consent. The location log generated by Safe Paths cannot be accessed from outside the user’s device, meaning data transfer occurs only if the user chooses to share it with a researcher or health official.
+## Multiple Capabilities
 
-<img align="right" src="./assets/PreviewUI.png" data-canonical-src="./assets/PreviewUI.png"/>
+Safe Paths is designed to support a range of DCT and public health use cases. Currently the main build uses GPS for location tracking. Our roadmap includes adding support for other location and proximity technologis, symptom tracking, and communication with PHAs.
```
Comment (Ferrumofomega) [inline README.md]: Safe Paths is designed to support a range of DCT and public health use cases. Currently the main build uses GPS for location tracking with GAEN BT tracing strategies in active development. Our roadmap includes adding support for other location and proximity technologies, symptom tracking, and communication with PHAs.

File: README.md [hunk 5]
```diff
@@ -9,47 +9,66 @@
...
+### Project Aurora and the Google Apple Exposure Notification API (GAEN) 
 
-Safe Paths is a ‘privacy-first’ app that allows you to log your GPS trails on your own phone. The information is stored locally and never shared with anyone (not even with us or MIT) until you explicitly decide to manually export the data.
+Project Aurora is the code name for a build from Safe Paths for an app that is fully compliant with GAEN. This build will not include GPS, and it will be compliant with all the rules and requirements for GAEN apps. [Learn more about Project Aurora.](https://covidsafepaths.org/a-new-open-source-solution-for-the-google-apple-exposure-notification-api/) 
 
-The location log generated by Safe Paths cannot be accessed from outside the user’s device. However, the user can import and export their location information and use it in other projects and applications.
+## Multiple Implementation Strategies 
 
-Safe Paths logs your device’s location once every five minutes and stores 14 days of data.
+Safe Paths is designed to support a variety of implementation strategies.
```
Comment (Ferrumofomega) [inline README.md]: The Safe Paths app is being developed to support a variety of build 'flavors' of the application around core health and tracing functionality. Reach out to our team to discuss creating a `flavor` for your use-case.

File: README.md [hunk 6]
```diff
@@ -9,47 +9,66 @@
...
 
-### Private Kit WhitePaper
+### Path Check Release of COVID Safe Paths
+Safe Paths is available as an app published by Path Check in the [Apple App Store](https://apps.apple.com/us/app/covid-safe-paths/id1508266966) and the [Google Play App Store](https://play.google.com/store/apps/details?id=org.pathcheck.covidsafepaths). Any authorized pubic health authority can use Safe Paths. 
 
-[Apps Gone Rogue: Maintaining Personal Privacy in an Epidemic](https://drive.google.com/file/d/1nwOR4drE3YdkCkyy_HBd6giQPPhLEkRc/view?usp=sharing)
+### Custom Builds 
 
-### Downloads for COVID Safe Paths
+We welcome public health authorities and other organizations implementing digital contact tracing strategies to create custom builds for their specific needs, incorporate Safe Paths features into their applications, or create downstream projects that stay linked to the Safe Paths project.
```
Comment (Ferrumofomega) [inline README.md]: We welcome public health authorities and other organizations implementing digital contact tracing strategies to create custom builds for their specific needs, incorporate Safe Paths features into their applications, or create downstream projects that stay linked to the Safe Paths project. If intending to fork the repository and develop off of it, be aware that this comes "at your own risk" for continued maintenance.

File: README.md [hunk 7]
```diff
@@ -9,47 +9,66 @@
...
+We welcome public health authorities and other organizations implementing digital contact tracing strategies to create custom builds for their specific needs, incorporate Safe Paths features into their applications, or create downstream projects that stay linked to the Safe Paths project. 
 
-[Google Play](https://play.google.com/store/apps/details?id=org.pathcheck.covidsafepaths) | [Apple Store](https://apps.apple.com/us/app/covid-safe-paths/id1508266966)
+### GAEN Instances
+
+The rules for GAEN require that each public health authority release their own application. The goal with Project Aurora is to suppor this deployment strategy. 
+
+## End-to-End System
+
+Safe Paths is designed to work with Safe Places, a tool for contact tracing teams to work with location data in the course of contact tracing interviews and to publish points of concern. Without having to sharing their own location history, a Safe Paths user can download the points of concern from their PHA to identify if they have had risk of an exposure.
```
Comment (Ferrumofomega) [inline README.md]: Safe Paths is designed to work with [Safe Places](https://github.com/Path-Check/safeplaces-frontend), a tool for contact tracing teams to work with location data in the course of contact tracing interviews and to publish points of concern. Without having to sharing their own location history, a Safe Paths user can download the points of concern from their PHA to identify if they have had risk of an exposure.

File: README.md (hunk 2 above)
Comment (Ferrumofomega) [inline README.md]: One of the important aspects of the Safe Paths open source project is that it's supported by a large community of volunteers in addition to the open source developer community. Spanning as diverse domains as product management, user research, cryptography, security, compliance, design, and videography more than 1,400 Path Check volunteers are working together to support Safe Paths and help drive adoption around the world.

**Review by Ferrumofomega — COMMENTED** [review]
Added a few comments, mostly spelling but a few other comments included too.

**Review by tstirrat — APPROVED** [review]

**Pull request merged**